src/app/lib/config/services/pidgin.js

```diff
@@ -58,7 +64,7 @@ const pidgin = {
         },
         500: {
           statusCode: '500',
-          title: 'Mistake',
+          title: 'Mistake - 500',
```

Comment:
Same with `- 500`, I don't think it should be here

Reply:
Updated